stable/appmesh-jaeger/values.yaml

```diff
@@ -20,6 +20,23 @@ resources:
     cpu: 100m
     memory: 256Mi
 
+  ## Prometheus server readiness and liveness probe initial delay and timeout
+  ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+  ##
+probes:
+  liveness:
+    initialDelaySeconds: 0
+    periodSeconds: 10
+    timeoutSeconds: 1
+    successThreshold: 1
+    failureThreshold: 3
+  readiness:
+    initialDelaySeconds: 0
+    periodSeconds: 10
+    timeoutSeconds: 1
+    successThreshold: 1
+    failureThreshold: 3
```

Comment:
Can you please also update the README configuration section [1] with new the params

[1] https://github.com/aws/eks-charts/blob/master/stable/appmesh-jaeger/README.md#configuration